channels_jsonrpc/jsonrpcconsumer.py

```diff
@@ -231,10 +243,11 @@ def raw_receive(self, message, **kwargs):
         :return:
         """
         content = '' if "text" not in message else message["text"]
-        result = self.__handle(content, message)
+        result, result_type = self.__handle(content, message)
 
-        # Send responce back
-        self.send(text=self._encode(result))
+        # Send responce back only if it is a call, not notification
+        if result_type == 0:
```

Comment:
Of course it should be done, but for now It is just a quick working solution. These lines would anywhere look different after moving to a JSON-RPC decoding/encoding library, and I plan to do it soon, because I need support of batch calls, and, may be, protocol versions 1.1/1.2 along with 2.0. So this is a temporary working solution.